README.md

```diff
@@ -1,101 +1,98 @@
 <p align="center">
-  <a href="https://www.axoniq.io/products/axon-framework">
-    <img src="https://www.axoniq.io/hubfs/axon-framework.svg" alt="Axon Framework logo" width="200" height="200">
+  <a href="https://axoniq.io/">
+    <img src="https://www.axoniq.io/hubfs/axoniq-light.svg" alt="Axon Framework logo" width="600" height="200">
   </a>
 </p>
 
-<h1 align="center">Axon Framework</h1>
+<h1 align="center"></h1>
 
 <p align="center">
-  Build modern event-driven systems with AxonIQ technology.
+  Build modern event-driven systems with AxonIQ technology
   <br>
-  <a href="https://www.axoniq.io/products/axon-framework"><strong>Product Description »</strong></a>
+  <a href="https://www.axoniq.io/"><strong>Learn more at our website »</strong></a>
   <br>
   <br>
-  <a href="https://github.com/AxonIQ/code-samples">Code Samples Repo</a>
+  <a href="https://www.axoniq.io/products/axon-framework">Axon Framework</a>
   ·
-  <a href="https://developer.axoniq.io/axon-framework/overview">Technical Overview</a>
+  <a href="https://www.axoniq.io/products/axon-server">Axon Server</a>
   ·
-  <a href="https://github.com/AxonFramework/AxonFramework/issues">Feature / Bug Request</a>
+  <a href="https://www.axoniq.io/products/axoniq-console">AxonIQ Console</a>
 
 </p>
 
+#
 
+<p><br/></p>
+<img src="https://www.axoniq.io/hubfs/axon-framework-line-light.png" alt="Axon Framework logo" width="500">
 
+The Axon Framework is an open source framework that's 100% Java and enables developers to build scalable and maintainable applications using a message-driven approach. It simplifies the complexities of developing distributed systems by providing a structured way to handle commands, events, and queries within your application. [If you're not familiar with the Event Sourcing pattern](https://www.axoniq.io/concepts/cqrs-and-event-sourcing), you can learn more about it on our website.
 
-# Axon Framework
-[![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.axonframework/axon/badge.svg)](https://maven-badges.herokuapp.com/maven-central/org.axonframework/axon)
-![Build Status](https://github.com/AxonFramework/AxonFramework/workflows/Axon%20Framework/badge.svg?branch=master)
-[![SonarCloud Status](https://sonarcloud.io/api/project_badges/measure?project=AxonFramework_AxonFramework&metric=alert_status)](https://sonarcloud.io/dashboard?id=AxonFramework_AxonFramework)
```

Comment:
I would really want these badges to be somewhere on the main `README.md`, as they quickly show the status of the repository.